types/resource.go

```diff
@@ -776,3 +776,15 @@ type RecurringJobSpec struct {
 type RecurringJobStatus struct {
 	OwnerID string `json:"ownerID"`
 }
+
+type SupportBundleSpec struct {
+	IssueURL    string `json:"issueURL"`
+	Description string `json:"description"`
+}
+
+type SupportBundleStatus struct {
+	State      string      `json:"state,omitempty"`
+	Filename   string      `json:"filename,omitempty"`
```

Comment:
Need to use CamelCase for Go variable name and camelCase for json.
`FileName: json fileName`
`FileSize: json fileSize`

Reply:
Updated